docs/user-feedback/README.md

```diff
@@ -0,0 +1,47 @@
+# User Feedback
+
+## Goals
+
+The Algorand Foundation is working on developing tooling to make it easier for developers to build on Algorand. One of the key components of this tooling is smart contract languages. The Algorand Foudnation has previously released Algorand Python, which enables developers to write smart contracts in Python. The compiler for Algorand Python is called Puya and can be leveraged by any other language. The next language being developed for Puya is Algorand TypeScript. The goal is Algorand TypeScript is to make it easy for TypeScript develoeprs to start building on Algorand. The full list of guiding pinciples can be found [here](../README.md#guiding-principals).
```

Comment:
I wonder if perhaps this debate might be fundamentally changed if the guiding principle is clarified to state if we are aiming for accessibility to _JavaScript_ developers and not just _TypeScript_ developers.  Given how flexible Python is as a language, if we want to aim for a similar degree of approachability, I actually think that perhaps we should be targeting familiarity to _JavaScript_ devs.  That might then substantively influence decisions on the design of the language.

Reply:
I feel like we are targeting JavaScript transitively through TypeScript. I hesitate to say JavaScript because we basically enforce typing, meaning you can't actually use raw JavaScript without the types.